### PR TITLE
Fixing alignment of settings close button

### PR DESCRIPTION
**Pull request by ryanml**
Popup:
Screen Shot 2021-09-23 at 9 03 02 PM

Responsive layout:
https://user-images.githubusercontent.com/8732757/134616537-273af6df-4405-44de-8a34-3b930078a53c.mov

### CODE REVIEW

File: ui/pages/settings/index.scss [hunk 1]
```diff
@@ -26,7 +26,6 @@
         text-overflow: ellipsis;
         overflow: hidden;
         white-space: nowrap;
-        max-width: 250px;
```
Comment (ryanml) [inline ui/pages/settings/index.scss:29]: @adonesky1 was there a case this was needed for in particular?

Comment (ryanml) [reply ui/pages/settings/index.scss:29]: Ah, now I see it served a purpose for very long contact names, so this needs to not be removed then but rather re-worked for normal use cases

Comment (adonesky1) [reply ui/pages/settings/index.scss:29]: yes I didn't do this elegantly before. But now removing it is causing thi
Screen Shot 2021-09-24 at 9 58 30 AM
Screen Shot 2021-09-24 at 9 58 24 AM
s:

Comment (ryanml) [reply ui/pages/settings/index.scss]: @adonesky1 I pushed changes that should cover all cases:
Screen Shot 2021-09-24 at 8 23 28 AM
Screen Shot 2021-09-24 at 8 23 03 AM
Screen Shot 2021-09-24 at 8 22 53 AM

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Review by darkwing — APPROVED** [review]

**Pull request merged**